src/leanback/java/com/archos/mediacenter/video/leanback/presenter/ShortcutDetailsPresenter.java

```diff
@@ -24,6 +24,10 @@
     protected void onBindDescription(ViewHolder viewHolder, Object item) {
         Shortcut shortcut = (Shortcut) item;
         viewHolder.getTitle().setText(shortcut.getName());
-        viewHolder.getSubtitle().setText(shortcut.getFriendlyUri());
+        
+        String uri = shortcut.getFriendlyUri();
+        String newUri = uri.substring(0, uri.length() - 1);
```

Comment:
perhaps better to remove specifically "/" instead of the systematically the last char

Reply:
Need to check if all the shortcuts smb/upnp/sftp etc.. has an ending /